packages/waku/src/lib/plugins/vite-plugin-deploy-cloudflare.ts

```diff
@@ -132,7 +138,7 @@ export function deployCloudflarePlugin(opts: {
         writeFileSync(
           workerEntrypoint,
```

Comment:
I wonder if index.js and serve-cloudflare.js can be combined.

Reply:
It could be. We could remove this and update SERVE_JS to `const SERVE_JS = 'index.js';`